tests/foreman/ui/test_hw_model.py

```diff
@@ -111,14 +114,20 @@ def test_positive_delete(self):
 
         @expectedresults: Hardware-Model is deleted
         """
+        def not_start_with_digit(value):
+            return not value[0].isdigit()
+
+        valid_names = valid_hw_model_names(
+            not_start_with_digit,
+            'QE Fake Hardware '
+        )
+
         with Session(self.browser) as session:
-            for test_data in valid_hw_model_names():
+            for test_data in valid_names:
+                if 'bugzilla' in test_data:
+                    # while bz 1265150 is not cherry picked html is not valid
+                    # input
+                    continue
```

Comment:
I'm a bit concerned that we will just forget to unskip this in the future when the bug is fixed. Not sure how to prevent this though. Maybe some message would be useful, so that guys could see that something is being skipped when analyzing test results.